entrypoints/webcomponents-hi-ce-sd-index.js

```diff
@@ -0,0 +1,22 @@
+/**
+@license
+Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+*/
+'use strict';
+
+/*
+ * Polyfills loaded: HTML Imports, Custom Elements, Shady DOM/Shady CSS
+ * Used in: Safari 9, Firefox, Edge
+ */
+
+import '../bower_components/html-imports/src/html-imports.js'
+import '../bower_components/custom-elements/custom-elements.min.js'
+import '../bower_components/shadydom/shadydom.min.js'
+import '../bower_components/shadycss/src/ShadyCSS.js'
```

Comment:
BTW, I just heard from @azakus that ShadyCSS comes in two parts, the file included here and `custom-style-element.js`.